packages/spor-react/src/card/StaticCard.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Box, useStyleConfig } from "@chakra-ui/react";
+/**
+ * Renders a static card.
+ *
+ * The most basic version looks like this:
+ *
+ * ```tsx
+ * <StaticCard colorScheme="white">
+ *   Content
+ * </StaticCard>
+ * ```
+ *
+ * Cards are not interactive by default. If you specify the `as` prop to be a link or a button, you can make them work as links or buttons respectively. This will also give it a drop shadow.
+ *
+ * ```tsx
+ * <StaticCard colorScheme="blue" as="button" onClick={handleClick}>
+ *   Click for profit
+ * </StaticCard>
+ * <StaticCard colorScheme="green" as="a" href="https://vy.no">
+ *   Go to start
+ * </StaticCard>
+ * ```
```

Comment:
Dette kan du fjerne – statiske kort får ikke lenger denne drop-shadowen. Men du kan kanskje legge inn et eksempel på hvordan du gjør et StaticCard til å rendre en `section` istedenfor?

```suggestion
 * Static cards can also be rendered as whatever DOM element you want – like a li (list item) or an article. You do this by specifying the `as` prop:
 *
 * ```tsx
 * <StaticCard colorScheme="green" as="section">
 *   This is now a <section /> element
 * </StaticCard>
 * ```
```